### PR TITLE
Menu navigation improvement

### PR DESCRIPTION
**Pull request by jota12x**
- [x] Add link to ML Search

- [x] Add sections to Top , Tools/Tutorials and Login (with Google Icon)

- [x] Styles according to Suzana

- Desktop:
imagen

- Ipad Pro:
imagen

- Ipad:

- Phone:
imagen

### CODE REVIEW

**Comment by aRhezaHarliman** [comment]
Thanks Juan! umm.. one request, would you mind to change the request into English? 😅

**Comment by rhezaharliman** [comment]
Thanks Juan! umm.. one request, would you mind to change the description into English? 😅

File: src/components/molecules/SidebarMenu.tsx [hunk 1]
```diff
@@ -22,41 +22,35 @@ const SidebarMenu: React.FunctionComponent<SidebarMenuProps> = ({
...
     </Menu.Item>
     <Menu.Item as="a" href="https://twitter.com/__mlt__" target="__blank">
       <Image
-        src={SLACK_LOGO}
+        src={GOOGLE_LOGO}
         width="14"
         style={{ display: 'inline-block', position: 'absolute', right: 18 }}
       />
-      Discuss
+      Login
```
Comment (rhezaharliman) [inline src/components/molecules/SidebarMenu.tsx:53]: Should it be in uppercase? 🤔

**Comment by jota12x** [comment]
@rhezaharliman fixed :)

**Pull request merged**